build.sbt

```diff
@@ -237,11 +236,11 @@ lazy val storehausLevelDB = module("leveldb").settings(
   fork in Test := true
 ).dependsOn(storehausCore % "test->test;compile->compile")
 
-lazy val storehausKafka08 = module("kafka-08").settings(
```

Comment:
this is going to change the jar name.

What do you think of having a kafka-09 module instead of ending support for 08? I actually don't know if anyone is using the 08 modules, but I hate to force an upgrade on someone or more realistically leave them stranded on an old version of storehaus.